ee/control/control.go

```diff
@@ -345,7 +345,7 @@ func (cs *ControlService) fetchAndUpdate(ctx context.Context, subsystem, hash st
 	// Consumer and subscriber(s) notified now
 	if err := cs.update(ctx, subsystem, data); err != nil {
 		// Returning the error so we don't store the hash and we can try again next time
-		slogger.Log(ctx, slog.LevelError,
+		slogger.Log(ctx, slog.LevelWarn,
```

Comment:
I thiiiiiink this is generally a warning, and if it's a true error then the subsystem itself should be logging it at the error level for reporting.